### PR TITLE
Interface for updating auth and MFA

### PR DESCRIPTION
**Pull request by ceslami**
@reustmd

### CODE REVIEW

**Comment by reustmd** [comment]
@ceslami updated with implementations.

I did them dumbly for now, duplicated all ze code. Try this out. If it works, and the implementations hold, then I'll remove the duplication and add some tests.

**Comment by ceslami** [comment]
:smiley: :smiley: :smiley:  nice!

File: src/main/java/com/plaid/client/DefaultPlaidUserClient.java [hunk 1]
```diff
@@ -85,15 +84,29 @@ public AccountsResponse achAuth(Credentials credentials, String type, ConnectOpt
...
     @Override
     public AccountsResponse mfaAuthStep(String mfa, String type) throws PlaidMfaException {
 
-        return handleMfa("/auth/step", mfa, type, AccountsResponse.class);
+        return handleMfaPost("/auth/step", mfa, type, AccountsResponse.class);
+    }
+
+    @Override
+    public AccountsResponse updateMfaAuthStep(String mfa, String type) throws PlaidMfaException {
+        return handleMfaPost("/auth/step", mfa, type, AccountsResponse.class);
```
Comment (ceslami) [inline src/main/java/com/plaid/client/DefaultPlaidUserClient.java]: patch?

Comment (ceslami) [inline src/main/java/com/plaid/client/DefaultPlaidUserClient.java]: nvm ill just pick this up from here

**Comment by ceslami** [comment]
@reustmd This works perfectly with my controller-level integration tests for both bank authentication and MFA. Nice job :+1: :zap:

**Comment by ceslami** [comment]
Cut a version and uploaded to nexus

**Pull request merged**